### ISSUE TITLE
unable to run succcessfully

### ISSUE THREAD

**Issue by pvssrikanth** [fastapi/full-stack-fastapi-template#373]
Dear Sir,
I am exited of this project 

But while running cookiecurter getting following errors Kindly help in resolution.

aefik_constraint_tag_staging [stag.iib.com]: 
traefik_public_constraint_tag [traefik-public]: 
flower_auth [admin:Loginiib@321]: 
sentry_dsn []: 
docker_image_prefix []: iib
docker_image_backend [iibbackend]: 
docker_image_celeryworker [iibceleryworker]: 
docker_image_frontend [iibfrontend]: 
  File "/tmp/tmpoGXJgy.py", line 4
    path: Path
        ^
SyntaxError: invalid syntax
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
iibadmin@iibadmin-uat-ubuntu:~/srik/projeccts$

**Comment by NMisko** [comment]
Can you check which python version you are using? You'd get this error when running it on python 2.

**Comment by pvssrikanth** [comment]
Dear Sir,
I am using pytho 3.9 in env
(fastadminenv) ***@***.***:~/srik/mayprojects/full-stack-fastapi-postgresql$
ls
CONTRIBUTING.md  cookiecutter.json  {{cookiecutter.project_slug}}  hooks
 img  LICENSE README.md  scripts
(fastadminenv) ***@***.***:~/srik/mayprojects/full-stack-fastapi-postgresql$
python -V
*Python 3.9.1*
(fastadminenv) ***@***.***
:~/srik/mayprojects/full-stack-fastapi-postgresql$

Can you suggest now how to run the cookiecutter command

Please advise i am novice but your project is really useful.

Thanking you ,
srikanth

**Comment by pvssrikanth** [comment]
Dear Sir,

(fastadminenv) ***@***.***:~/srik/mayprojects/full-stack-fastapi-postgresql$
sudo cookiecutter https://github.com/tiangolo/full-stack-fastapi-postgresql
You've downloaded
/home/iibadmin/.cookiecutters/full-stack-fastapi-postgresql before. Is it
okay to delete and re-download it? [yes]: yes
project_name [Base Project]: iib
project_slug [iib]:
domain_main [iib.com]:
domain_staging [stag.iib.com]:
docker_swarm_stack_name_main [iib-com]:
docker_swarm_stack_name_staging [stag-iib-com]:
secret_key [changethis]:
first_superuser ***@***.***:
first_superuser_password [changethis]:
backend_cors_origins [["http://localhost", "http://localhost:4200", "
http://localhost:3000", "http://localhost:8080", "https://localhost", "
https://localhost:4200", "https://localhost:3000", "https://localhost:8080",
"http://dev.iib.com", "https://stag.iib.com", "https://iib.com", "
http://local.dockertoolbox.tiangolo.com", "http://localhost.tiangolo.com"]]:

smtp_port [587]:
smtp_host []:
smtp_user []:
smtp_password []:
smtp_emails_from_email ***@***.***:
postgres_password [changethis]:
pgadmin_default_user ***@***.***:
pgadmin_default_user_password [changethis]:
traefik_constraint_tag [iib.com]:
traefik_constraint_tag_staging [stag.iib.com]:
traefik_public_constraint_tag [traefik-public]:
flower_auth [admin:changethis]:
sentry_dsn []:
docker_image_prefix []:
docker_image_backend [backend]:
docker_image_celeryworker [celeryworker]:
docker_image_frontend [frontend]:
  File "/tmp/tmp3EjvwH.py", line 4
    path: Path
        ^
SyntaxError: invalid syntax
ERROR: Stopping generation because post_gen_project hook script didn't exit
successfully
Hook script failed (exit status: 1)
(fastadminenv) ***@***.***
:~/srik/mayprojects/full-stack-fastapi-postgresql$


I DONNOT KNOW HOW TO SETUP THIS UP AND RUNNING.

Kindly help in resolution

**Comment by NMisko** [comment]
What does `cookiecutter -V` return?

**Comment by pvssrikanth** [comment]
Hi Sir,

It returned below

app.py envmail  fastadminenv  full-stack-fastapi-postgresql  __pycache__
 testing  testmail.py
***@***.***:~/srik/mayprojects$ source
fastadminenv/bin/activate
(fastadminenv) ***@***.***:~/srik/mayprojects$ python -V
Python 3.9.1
(fastadminenv) ***@***.***:~/srik/mayprojects$
* cookiecutter -VCookiecutter 1.6.0 from /usr/lib/python2.7/dist-packages
(Python 2.7)*
(fastadminenv) ***@***.***:~/srik/mayprojects$

Please advise how to fix this issue.

Thanks & Regards
srikanth

**Comment by NMisko** [comment]
Yeah, so cookiecutter was installed outside your python 3 environment. You can call `python -m 'pip install cookiecutter'` to make sure you're using the correct `pip`. Alternatively: leave the environment (`deactivate`), then uninstall the python 2 version `pip uninstall cookiecutter` then install the python 3 version `pip3 install cookiecutter`.

**Comment by pvssrikanth** [comment]
Dear Sir,
Thanks for your prompt response.

I am novice here

Trying top run the usggested cocmmands But not working

env) ***@***.***:~/srik/mayprojects$ python -V
Python 3.9.1
(env) ***@***.***:~/srik/mayprojects$ python3.9 -m 'pip
install cookiecutter'
/home/iibadmin/srik/mayprojects/env/bin/python3.9: No module named pip
install cookiecutter
(env) ***@***.***:~/srik/mayprojects$ *python -m 'pip
install cookiecutter'*
/home/iibadmin/srik/mayprojects/env/bin/python: No module named pip install
cookiecutter
(env) ***@***.***:~/srik/mayprojects$
Please guide again

Thanks

**Comment by NMisko** [comment]
Try it without the quotation marks, sorry: `python3 -m pip install cookiecutter`

**Comment by pvssrikanth** [comment]
Thanks Sir
I will try tomo and let y know

Thanks for your excellent support and guidance

Regards
Srikanth

**Comment by pvssrikanth** [comment]
Dear Sir,
Now Cookiecutter setup with  python3.9 However i am still facing issue

pgadmin_default_user ***@***.***:
pgadmin_default_user_password [changethis]:
traefik_constraint_tag [iib.com]:
traefik_constraint_tag_staging [stag.iib.com]:
traefik_public_constraint_tag [traefik-public]:
flower_auth [admin:changethis]:
sentry_dsn []: iib
docker_image_prefix []: iib
docker_image_backend [iibbackend]:
docker_image_celeryworker [iibceleryworker]:
docker_image_frontend [iibfrontend]:





* File "/tmp/tmplrmymn.py", line 4    path: Path        ^SyntaxError:
invalid syntaxERROR: Stopping generation because post_gen_project hook
script didn't exit successfullyHook script failed (exit status: 1)*
(env) ***@***.***:~/srik/fullstackfastapi$ cookiecutter -V
Cookiecutter 1.7.2 from
/home/iibadmin/srik/fullstackfastapi/env/lib/python3.9/site-packages
(Python 3.9)
(env) ***@***.***:~/srik/fullstackfastapi$

 Kindly  help in resolving this issue.

Regards,
srikanth

**Comment by pvssrikanth** [comment]
Dear Sir,
This is really very exciting project.

Kindly help me in resolving this issue.

I want to see its successful build and see the demo

so kindly help

**Comment by pvssrikanth** [comment]
Unable  to run this and test.

kindly help

**Comment by YuriiMotov** [comment]
Outdated
